.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+---
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "npm"
+    directory: "/"
+    open-pull-requests-limit: 20
```

Comment:
Let's set this one to 2 initially.

Reply:
I would try 5, 2 is too small for NPM and original 20 was too much